### PR TITLE
ArubaOS - fix client count polling, add ap count polling.

### PR DESCRIPTION
**Pull request by twilley**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/arubaos.yaml [hunk 1]
```diff
@@ -3,8 +3,10 @@ text: ArubaOS
 type: wireless
 icon: aruba
 over:
-    - { graph: device_arubacontroller_numaps, text: 'Number of APs' }
-    - { graph: device_arubacontroller_numclients, text: 'Number of Clients' }
+    - { graph: device_wireless_ap-count, text: 'AP Count' }
+    - { graph: device_wireless_clients, text: 'Client Count' }
+    - { graph: device_arubacontroller_numaps, text: 'Number of APs (Legacy)' }
```
Comment (murrant) [inline includes/definitions/arubaos.yaml]: Is this worthwhile to leave?  Maybe we should have a check if the rrd exists before displaying the overview graph?

Comment (PipoCanaja) [reply includes/definitions/arubaos.yaml]: @murrant That would be nice indeed !

**Comment by kkrumm1** [comment]
I did a test on this with my controller. Worked great!

File: includes/definitions/arubaos.yaml (hunk 1 above)
Comment (murrant) [reply includes/definitions/arubaos.yaml]: I took a look and it is not feasible with the current code. :(  Would take some major re-writing of graphing.

**Pull request merged**